README.md

```
@@ -1 +1,166 @@
+- Essential information only
+- Same styling as main CV
+
+## Publication Management
+
+### Adding New Publications
+
+1. **Main CV**: Add to beginning of enumerated list in `Vatsal_CV.tex`
+2. **Short CV**: Update with the 7 most recent publications (chronologically, not by impact)
+3. **Bibliography**: Add BibTeX entry to `mypublications.bib`
+
+### Selection Criteria for Short CV
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)